oslc_prolog/lib/oslc_dispatch.pl

```diff
@@ -11,10 +11,16 @@
   delete_handler/2,
   dispatch/1,
   response/1,
-  response/2
+  response/2,
+  select_acceptable_content_type/2
 ]).
 
 :- use_module(library(semweb/rdf11)).
+:- use_module(library(semweb/turtle)).
+
+:- multifile serializer/2.
```

Comment:
FEATURE: Users can extend supported MIME types by specifying custom `serializer/2` and `serialize_response/3`